contracts/Fellowship.sol

```diff
@@ -176,8 +179,9 @@ contract Fellowship {
 
     function setStakeAmount(uint256 _amount) external onlyRivendale {
         stakeAmount = _amount;
-        for(uint256 i = 0;i<fellowship.length;i++){
-            if(walkers[fellowship[i]].balance < stakeAmount){
+        // Should we also automatically set unfunded members as active? Or this should be a more "active" action?

```

Comment:
I don't think this will be much of an issue, so I was even debating taking it out, but if you do raise the stake amount for a potential user, they'd want to know that they guy actually has money staked